### ISSUE TITLE
[FEATURE] Valheim umod support

### ISSUE THREAD

**Issue by cyberlimbo** [GameServerManagers/LinuxGSM#3245]
Can you guys add umod support to Valheim?
It is already available (manual install).
Info: https://umod.org/games/valheim/guide

#keepyourgoodwork

**Comment by cyberlimbo** [comment]
up

**Comment by h3o66** [comment]
I would not integrate this till they have a prebuild version.
Till this happens, I would put this on hold.

**Comment by cyberlimbo** [comment]
They already have a functional umod version (at least i'm using the latest version and seems to be stable) ;)

**Comment by h3o66** [comment]
yeah, we would require a pre build version on our side, we do not want to hassle more around with new problems what could come up with a manual build.

There was also a little discussion on umod side:
https://umod.org/community/valheim/31876-no-way-to-install-umod-without-using-the-agent

**Comment by h3o66** [comment]
As they moved to a seperate install:
https://umod.org/guides/getting-started/installation

I would not integrate this, as they got a seperate installer this and dependencys, that might need additional resources to keep this up2date.

@dgibbs64 your opionion on this ?
